apps/nextra/docs.config.js

```diff
@@ -131,6 +131,153 @@ export const i18nConfig = Object.freeze({
+    blockchainSectionHeadline: "Discover blockchain features on Aptos",
+    performanceLabel: "Performance",
+    performanceDescription:
+      "Redefine blockchain performance with high TPS and low latency",
+    parallelExecutionLabel: "Parallel Execution",
+    parallelExecutionDescription:
+      "Learn how Block-STM powers parallelization and optimistic concurrency on Aptos",
+    validatorsLabel: "Validators & Fullnodes",
+    validatorsDescription:
+      "Learn more about what it takes to run a validator or fullnode",
+    keylessLabel: "Keyless",
+    keylessDescription: "Enjoy seamless user authentication with Keyless",
+    passkeysLabel: "Passkeys",
+    passkeysDescription: "Onboard seamlessly using biometrics",
+    randomnessLabel: "On-Chain Randomness",
+    randomnessDescription: "Unlock true fairness in gaming and beyond",
+    feePayerLabel: "Fee Payer",
+    feePayerDescription:
+      "Empower your transactions with seamless execution, letting others foot the bill",
+    multiSigLabel: "Multi-sig",
+    multiSigDescription:
+      "Secure your assets like never before with collaborative control and unparalleled safety",
+    gasLabel: "Gas",
+    gasDescription: "Trade smarter, not harder with low gas fees",
+    consensusLabel: "Consensus",
+    consensusDescription: "Learn more about DAG based consensus",
+    storageLabel: "Storage",
+    storageDescription:
+      "Learn more about Aptos' Jellyfish Merkle Tree and custom RocksDB configuration",
+    networkingLabel: "Networking",
+    networkingDescription:
+      "Learn about the topology of the Aptos blockchain and how nodes communicate with each other",
+    mempoolLabel: "Mempool",
+    mempoolDescription:
+      "Learn how transactions are buffered for processing before they're sent upstream",
+    stateSyncLabel: "State Sync",
+    stateSyncDescription:
+      "Learn how nodes downstream synchronizes with the latest state",
+
+    // Testimonials Section
+    testimonialsSectionHeadline: "Hear From The Ecosystem",
+    testimonial1Body:
+      "Building on Aptos has been transformative, offering unmatched transaction speed and reliability. We love the vibrant community and overall direction that the Aptos ecosystem is heading.",
+    testimonial1Author: "–Muhd Dahlan, CEO Zabava Labs",
+    testimonial2Body:
+      "Aptin Finance drives secure, intuitive, and seamless financial interactions for Aptos users. Aligned with Aptos' vision, we aim to establish a nexus of Omni-chain DeFi opportunities for a diverse global community.'",
+    testimonial2Author: "–Elvis Woo, CTO & Co-Founder, Aptin Finance",
+    testimonial3Body:
+      "Cellana Finance chose to build on Aptos because of its rapid transaction speed, cost-effectiveness, and robust security. Moreover, Aptos’s instant finality (less than 1s) will be a significant strength in attracting a variety of financial applications from Web 2 in the future.",
+    testimonial3Author: "–Andy Hoang, CEO, Cellana",
```

Comment:
Would you mind defining this as its own variable above so we can reuse for english and chinese?

Reply:
To some extent, this makes sense, but unfortunately, we might need TypeScript to annotate the current field types, yet the current configuration is all in JavaScript.

I think we can proceed with the current approach for now and improve upon it later.

I've seen that the Nextra framework has some documentation on multi-language support. 

Perhaps we can explore more convenient ways for reuse and synchronized updates across multiple languages in the future:
https://nextra.site/docs/guide/i18n

Reply:
Responded in dm but the docs above are for Nextra v2. For Nextra v3, they're using locale based folders, see here: https://the-guild.dev/blog/nextra-3